plugins/python-build/bin/python-build

```diff
@@ -899,7 +899,7 @@ build_package_micropython() {
     "$MAKE" $MAKE_OPTS
     cd ../ports/unix
     "$MAKE" $MAKE_OPTS axtls
-    "$MAKE" $MAKE_OPTS CFLAGS_EXTRA="-DMICROPY_PY_SYS_PATH_DEFAULT='\"${PREFIX_PATH}/lib/micropython\"' $CFLAGS_EXTRA"
+    "$MAKE" $MAKE_OPTS CFLAGS_EXTRA="-DMICROPY_PY_SYS_PATH_DEFAULT='\".frozen:${PREFIX_PATH}/lib/micropython\"' $CFLAGS_EXTRA"
```

Comment:
Won't this break previous versions?

If yes, the easiest way would probably be to set some kind of flag and do the corresponding thing depending on it. One way is to introduce another `build_*` function that would be setting it.

Reply:
I checked, it doesn't break previous versions:

```
$ micropython
MicroPython v1.17 on 2022-08-30; linux version
Use Ctrl-D to exit, Ctrl-E for paste mode
>>> import sys
>>> sys.path
['', '.frozen', '/home/serdyuk/.pyenv/versions/micropython-1.17/lib/micropython']
>>> import upip
>>>
```